themes/material/mfa/low-on-backup-codes.php

```diff
@@ -27,9 +27,10 @@
                     </h1>
                 </div>
 
+
                 <div class="mdl-card__title center" >
                     <p class="mdl-card__subtitle-text">
-                        <?= $this->t('{material:mfa:running_out_info}', ['{numBackupCodesRemaining}' => (int)$this->data['numBackupCodesRemaining']]) ?>
+                        <?= $this->t('{material:mfa:running_out_info}', ['{numBackupCodesRemaining}' => (string)(int)$this->data['numBackupCodesRemaining']]) ?>
```

Comment:
We might be able to remove both type casts here. It's currently casting to an `int`, then to ` string`, and I suspect it started as a string originally.
```suggestion
                        <?= $this->t('{material:mfa:running_out_info}', ['{numBackupCodesRemaining}' => $this->data['numBackupCodesRemaining']]) ?>
```

Reply:
Ahh, okay. Thanks.